### ISSUE TITLE
Uncaught TypeError: event.target.getModel is not a function

### ISSUE THREAD

**Issue by aaronmcohen** [stan3/gherkin-table-formatter#3]
[Enter steps to reproduce:]

1. Open Gherkin file with unicode in table
2. Run gherkin-table-formatter

**Atom**: 1.19.0-beta4 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [gherkin-table-formatter](https://github.com/stan3/gherkin-table-formatter) package 0.0.2

### Stack Trace

Uncaught TypeError: event.target.getModel is not a function

```
At /Users/aaroncohen/.atom/packages/gherkin-table-formatter/lib/gherkin-table-formatter.coffee:25

TypeError: event.target.getModel is not a function
    at /packages/gherkin-table-formatter/lib/gherkin-table-formatter.coffee:25:47)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom Beta.app/Contents/Resources/app/src/atom-environment.js:1338:34)
    at EventEmitter.outerCallback (/Applications/Atom Beta.app/Contents/Resources/app/src/application-delegate.js:334:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -2:28.9.0 core:save (input.hidden-input)
     -2:16.4.0 editor:newline (input.hidden-input)
     -2:10.6.0 core:copy (input.hidden-input)
     -2:08.7.0 core:paste (input.hidden-input)
     -2:08.4.0 editor:newline (input.hidden-input)
     -2:07.0 core:undo (input.hidden-input)
     -2:06.7.0 core:save (input.hidden-input)
     -1:58.6.0 core:move-up (input.hidden-input)
     -1:58.4.0 core:move-right (input.hidden-input)
     -1:57.9.0 core:select-down (input.hidden-input)
     -1:54.3.0 core:backspace (input.hidden-input)
     -1:52.5.0 core:save (input.hidden-input)
     -0:54 core:paste (input.hidden-input)
  2x -0:45.2.0 core:backspace (input.hidden-input)
     -0:41.2.0 core:confirm (input.hidden-input)
     -0:08.4.0 gherkin-table-formatter:format (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.4 
auto-detect-indentation 1.3.0 
busy-signal 1.4.3 
docker 0.7.0 
file-icons 2.1.9 
formatter-gofmt 1.0.1 
gherkin-autocomplete 2.0.0 
gherkin-table-formatter 0.0.2 
go-debug 1.5.0 
go-imports 0.1.1 
go-plus 5.5.5 
go-signature-statusbar 1.2.1 
hyperclick 0.1.2 
intentions 1.1.2 
language-ansible 0.2.1 
language-docker 1.1.8 
language-gherkin 1.0.4 
linter 2.2.0 
linter-ansible-linting 1.3.0 
linter-docker 0.2.0 
linter-markdown 4.1.0 
linter-ui-default 1.6.2 
markdown-preview-plus 2.4.10 
platformio-ide-terminal 2.5.3 
```

**Comment by emz-1** [comment]
Any solution's for this..?

**Comment by aaronmcohen** [comment]
Still running into this.

**Comment by ssnyder** [comment]
I'm also getting this

**Comment by bigshellltd** [comment]
Same for me

**Comment by kingbuzzman** [comment]
bump.

**Comment by coughe** [comment]
I changed the activate in my local gherkin-table-formatter.coffee to be:
```
    activate: ->
      @tableFormatter = new TableFormatter()
      #Register command to workspace
      @command = atom.commands.add 'atom-text-editor',
        'gherkin-table-formatter:format': (event) =>
          editor = event.currentTarget.getModel()
          @tableFormatter.format(editor)
```

It works for me now.  @stan3 fyi

**Comment by ghost** [comment]
This modification isn't running to me.

**Comment by blueacker** [comment]
Thanks @coughe , it works for me: Atom(1.23.2X64 high sierra), gherkin-table-formatter(0.0.2).
It's clearer that just changing the word "target" to "currentTarget"  in line 25.

The result is as following:
```
activate: ->
      @tableFormatter = new TableFormatter()
      #Register command to workspace
      @command = atom.commands.add "atom-text-editor",
        "gherkin-table-formatter:format", (event) =>
          @tableFormatter.format(event.currentTarget.getModel())
```

**Comment by roniyaniv** [comment]
Doesn't work for me (changed "target" to "currentTarget").

**Comment by tbhi** [comment]
believe fixed with 0.1.0
